tiltak/src/main/kotlin/no/nav/amt/tiltak/deltaker/service/DeltakerServiceImpl.kt

```diff
@@ -47,14 +46,12 @@ open class DeltakerServiceImpl(
 
 	override fun upsertDeltaker(personIdent: String, deltaker: DeltakerUpsert) {
 		val lagretDeltaker = hentDeltaker(deltaker.id)
-
-		transactionTemplate.executeWithoutResult {
-			if (lagretDeltaker == null) {
-				insertDeltaker(personIdent, deltaker)
-			} else if(!deltaker.compareTo(lagretDeltaker)){
-				update(deltaker)
-			}
-
+		val brukerId = brukerService.getOrCreate(personIdent)
```

Comment:
Er det ingen risiko for at noe blir usynk uten en transaksjon her det er veldig mye som ikke?

Reply:
Jeg tror ikke det er nødvendig/ønskelig med en transaksjon fordi:
1. En bruker kan opprettes og så krasjer noe senere og det er helt ok at endringen ikke rulles tilbake
2. En deltaker kan opprettes/endres og så krasjer det og det vil ikke gjøre noe, trenger ikke rulle tilbake
3. Status på deltaker kan endres og krasje men har ikke noen hensikt at de andre tingene over rulles tilbake